sphinx-doc/module-hoomd-logger.rst

```diff
@@ -0,0 +1,16 @@
+hoomd.logger
+--------------
+
+.. rubric:: Overview
+
+.. autosummary::
+    :nosignatures:
+
+    hoomd.logger.LoggerQuantity
+
+.. rubric:: Details
+
+.. automodule:: hoomd.logger
```

Comment:
We are documenting the `logger` module now, so users can use `Logger` from here.